scripts/pre_commit_linter.py

```diff
@@ -1011,7 +1011,7 @@ def _check_docstrings(all_files):
                     prev_line = file_content[line_num - 1].lstrip().rstrip()
 
                 # Check for space after """ in the docstring
-                if line.startswith('""" '):
+                if line.startswith('""" ') and line.endswith('"""'):
```

Comment:
This will only work for single line docstrings. What if the docstring is multi-line? You need to take care of that too, right?

Reply:
Yeah. I have corrected the same.

Reply:
Hmm, I was thinking of `line.startswith('""" ') or line.endswith(' """')` since we don't want doc lines that have a space between the starting quotes and the first word nor quotes that have a space in front of it that does not serve for indentation.

Reply:
@son-vyas, comments here?